src/styles/themes/default.js

```diff
@@ -97,6 +99,8 @@ const oldTheme = {
     placeholderText: colors.gray3,
     heroCard: colors.blue,
     uploadPreviewActivityIndicator: colors.gray1,
+    dropUIBG: 'rgba(6,27,9,0.92)',
```

Comment:
I don't think this is correct, but we're not using the old theme for now anyways, so NAB.